msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -341,7 +341,7 @@ class NativeAuthPublicClientApplication(
                     return@withContext SignInError(
                         errorType = ErrorTypes.INVALID_USERNAME,
                         errorMessage = "Empty or blank username",
-                        correlationId = "UNSET"
+                        correlationId = ""
```

Comment:
Why not set it at all, meaning the value will be null?

Reply:
The correlationId parameter is mandatory. Setting it to “” would be much easier than dealing with if null.